deployment/monocular/templates/ui-config.yaml

```diff
@@ -13,6 +13,9 @@ data:
       overrides: {
         googleAnalyticsId: '{{.Values.ui.googleAnalyticsId}}',
         appName: '{{.Values.ui.appName}}',
+        {{- if .Values.ui.backendHostname }}
+        backendHostname: '{{ .Values.ui.backendHostname }}',
```

Comment:
I decided not to make the default `/api` to not change the way it currently work, but maybe that makes sense?